packages/lynx-devtool-web/src/containers/TestBench/TestBench.tsx

```diff
@@ -0,0 +1,327 @@
+            }}
+          >
+            Clear
+          </Button>
+        ) : (
+          <span />
+        )}
+      </>
+    );
+  };
+
+  const renderResultView = () => {
+    const introduction = '\nIf you had opened page before starting record,the possible reasons are as follows:\n';
+    const aboveLynx2_11Hint =
+      '- If the page is loaded when the APP is started, you can try TestBench starting recording function;\n';
+    const generalHint = '- Clear the APP cache, restart the APP and start recording;';
+
+    const lynxVersion = selectedDevice.info?.sdkVersion ?? 'unknown';
+    const columns = [
+      {
+        title: 'Id',
+        dataIndex: 'id',
+        key: 'id'
+      },
+      {
+        title: 'Device',
+        dataIndex: 'deviceModel',
+        key: 'deviceModel'
+      },
+      {
+        title: 'App',
+        dataIndex: 'appName',
+        key: 'appName'
+      },
+      {
+        title: 'Status',
+        dataIndex: 'status',
+        key: 'status',
+        render: (_: any, { isValid, message }: any) => {
+          if (isValid) {
+            return (
+              <div className="status-success">
+                <CheckOutlined />
+              </div>
+            );
+          } else {
+            let hint = message;
+            if (!(lynxVersion === 'unknown')) {
+              const indexForFirstDot = lynxVersion.indexOf('.');
+              const indexForSecondDot = lynxVersion.indexOf('.', indexForFirstDot + 1);
+              const version = parseFloat(lynxVersion.substring(0, indexForSecondDot));
+              if (version >= 2.2 && version < 2.11) {
+                hint = message + introduction + generalHint;
+              } else if (version >= 2.11) {
+                hint = message + introduction + aboveLynx2_11Hint + generalHint;
+              }
+            }
+
+            return (
+              <div className="status-failure">
+                <CloseOutlined />
+                <span style={{ display: 'inline-block', whiteSpace: 'pre-line' }}>{hint}</span>
+              </div>
+            );
+          }
+        }
+      },
+      {
+        title: 'Preview',
+        dataIndex: 'pic',
+        key: 'pic',
+        render: (_: any, { pic }: any) => {
+          return (
+            <Image
+              style={{
+                maxWidth: 200,
+                maxHeight: 200,
+                border: 'solid',
+                borderRadius: 2,
+                borderWidth: 1,
+                borderColor: 'rgb(240,240,240)'
+              }}
+              src={pic}
+              preview={{ src: pic }}
+            />
+          );
+        }
+      },
+      {
+        title: 'QRCode',
+        dataIndex: 'url',
+        key: 'url',
+        render: (_: any, { url }: any) => {
+          return (
+            <Popover
+              content={
+                <div className="testbentch-qrcode-pop">
+                  <div>Use the LynxExample App to scan the QR code</div>
```

Comment:
There's no LynxExample App

Reply:
And we are not sure what form Lynx DevTool connects to the host application through web socket in the future.